modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -1815,3 +1816,46 @@ translationAndAutosaveTest(
 		).toHaveText('Approved');
 	}
 );
+
+baseTest(
+	'Verify Journal Article ID field is not editable after assignment',
+	{
+		tag: '@LPD-39034',
+	},
+	async ({journalEditArticlePage, page, site}) => {
+		const systemSettingsPage = new SystemSettingsPage(page);
+		await systemSettingsPage.goToSystemSetting(
+			'Web Content',
+			'Administration'
+		);
+
+		const toggleSelector =
+			'label:has-text("Journal Article Force Autogenerate ID") >> input[type="checkbox"]';
+		const toggle = page.locator(toggleSelector);
+		if (await toggle.isChecked()) {
+			await toggle.click();
+		}
```

Comment:
This isn't need use directly [toggle.check()](https://playwright.dev/docs/api/class-locator#locator-check)